### PR TITLE
add how to generate self-signed certificates

### PR DESCRIPTION
**Pull request by eisoku9618**
I always forget the command to generate self-signed certificates, so I'm happy if README.md teaches me it.

### CODE REVIEW

File: examples/compose/README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ change hostname set in this file, and also in the `nginx/nginx.conf` file.
 This example is set up in a way so you can use self-signed certificates. Of
 course this is not something you would want to do in production, but this way we
 ease up the task for those who are curious to try it out.
+```bash
+openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout portus.key -out portus.crt
+```
```
Comment (mssola) [inline examples/compose/README.md]: I'd add some introductory text, so the whole section would look like as follows:

This example is set up in a way so you can use self-signed certificates. Of course this is not something you would want to do in production, but this way we ease up the task for those who are curious to try it out. In order to create self-signed certificates, you could use the following command:

```bash
openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout portus.key -out portus.crt
```

After that, you can simply move the `portus.key` and the `portus.crt` files into the `secrets` directory.

**Comment by mssola** [comment]
Also, remember to sign off your work (use the `-s` flag of `git commit`) :wink:

File: examples/compose/README.md (hunk 1 above)
Comment (eisoku9618) [reply examples/compose/README.md]: Thanks! Updated.

**Comment by mssola** [comment]
@eisoku9618 thanks for this :+1: Now, could you squash your commits into a single one, and then rebase it on top of current master? After that, I'll gladly merge it :yum:

**Pull request merged**